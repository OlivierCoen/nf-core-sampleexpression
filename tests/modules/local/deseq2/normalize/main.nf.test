nextflow_process {

    name "Test Process DESEQ2_NORMALIZE"
    script "modules/local/deseq2/normalize/main.nf"
    process "DESEQ2_NORMALIZE"
    tag "deseq2_normalize"

    test("Should run without failures") {

        when {

            process {
                """
                input[0] = file('$baseDir/tests/input/normalize/all_counts.csv')
                """
            }
        }

        then {
            assert process.success
            assert snapshot(process.out.csv).match()
        }

    }

}
