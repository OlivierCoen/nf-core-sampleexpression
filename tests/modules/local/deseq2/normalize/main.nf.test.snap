{
    "Should run without failures": {
        "content": [
            [
                "all_counts_normalized.csv:md5,b999d6fd44fb54c8b311d7e84d64ce9f"
            ]
        ],
        "meta": {
            "nf-test": "0.9.0",
            "nextflow": "24.04.4"
        },
        "timestamp": "2024-10-17T15:03:47.124237706"
    }
}